packages/utils/src/Pending.ts

```diff
@@ -1,23 +1,33 @@
 import PromiseUtils from './PromiseUtils';
 
+interface CancelablePromise<T> extends Promise<T> {
+  cancel: () => void;
+}
+
```

Comment:
Import this from PromiseUtils since they're referencing the same interface really